### PR TITLE
Simple APNS job to test apple push notifications 

### PR DESCRIPTION
**Pull request by koomar**
- next step is to inject cert using environment variable

### CODE REVIEW

File: simple_scheduler/jobs/apns_job.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import logging
+
+from ndscheduler import job
+from apns import APNs, Payload
+
+logger = logging.getLogger(__name__)
+
+
+class APNSJob(job.JobBase):
+    TIMEOUT = 10
```
Comment (koomar) [inline simple_scheduler/jobs/apns_job.py]: remove TIMEOUT

File: simple_scheduler/jobs/apns_job.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                # APNS Title's Alert Text
+                {'alert': 'string', 'description': 'What do you want to send?'},
+            ],
+            'example_arguments': ('["da1232badh2", "Hello from scheduler"]')
+        }
+
+    def run(self, token, alert="Hello World",  *args, **kwargs):
+        print 'Sending %s to %s' % (alert, token)
+
+        apns = APNs(use_sandbox=False, cert_file='simple_scheduler/jobs/apns-cert.pem')
```
Comment (wenbinf) [inline simple_scheduler/jobs/apns_job.py]: Using environment variable here can be as easy as:

cert_file = os.environ['APNS_CERT_PATH'] or 'simple_scheduler/jobs/apns-cert.pem'
apns = APNs(use_sandbox=False, cert_file=cert_file)

Then in the above 'notes', add something like "the environment variable APNS_CERT_PATH" should be provided for apns cert file".

File: simple_scheduler/jobs/apns_job.py (hunk 1 above)
Comment (koomar) [inline simple_scheduler/jobs/apns_job.py]: Done

File: simple_scheduler/jobs/apns_job.py (hunk 2 above)
Comment (koomar) [inline simple_scheduler/jobs/apns_job.py]: Done, thanks

**Comment by wenbinf** [comment]
:+1:

**Pull request merged**